app/src/main/java/com/teamagam/gimelgimel/app/common/SynchronizedDataHolder.java

```diff
@@ -1,4 +1,4 @@
-package com.teamagam.gimelgimel.app.view.viewer.cesium;
+package com.teamagam.gimelgimel.app.common;
```

Comment:
I'm not sure that common is the right name. we have utils and also com.teamagam.gimelgimel.helpers (_autodesk).